avocado/utils/configure_network.py

```diff
@@ -75,6 +75,53 @@ def set_ip(ipaddr, netmask, interface):
             raise NWException("ifup fails: %s" % ex)
 
 
+def set_ipv6(ipaddr, netmask, interface):
```

Comment:
can this be achieved in `set_ip` module as well having an extra argument where it decide as it is ipv4 or ipv6

Reply:
Resending the patch by enabling ipv6 in the existing function  set_ip(ipaddr, netmask, interface)
IPV6 is enabled in all the cases when this utility is being called. Test can proceed when both ipv4 and ipv6 address are assigned, so following this standard.